content/themis/languages/python/installation.md

```diff
@@ -75,13 +75,151 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
 If the stable package version does not suit your needs,
 you can manually build and install the latest version of Themis from source code.
 
+ 1. [Build and install Themis Core library](/themis/installation/installation-from-sources/)
+    into your system.
+
+ 2. Install Python dependencies
+
+    Depending on how you want to install PyThemis, there are different requirements.
```

Comment:
Let's list the available approaches. One into the virtualenv and one system-wide installed as system package. Because further we describe steps how install dependencies, but not obvious for what. And why the user needs .whl. Usually developers installs packages with commands like `pip install` or `python setup.py` and they don't care what the files are generated. They interested in the package installed into their local or system environment. So we should clarify that we support 2 approaches. Local installation into venv and you need next dependencies, or installation into the system using system package manager and you need next dependencies.